Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -331,9 +331,11 @@ extension NSManagedObjectContext {
         }
 
         // Note: membership payload should be saved before all the members
+        var membershipData: (dto: MemberDTO, payload: MemberPayload)?
```

Comment:
I will remove it when backend change has been deployed, currently it just acts as a workround